app.js

```diff
@@ -28,7 +14,6 @@ var express = require('express'),
 var favicon = require('serve-favicon'),
     cookieSession = require('cookie-session'),
     cookieParser = require('cookie-parser'),
-    logger = require('morgan'),
```

Comment:
the logs should still show up with `console.log` no? we just aren't formatting it in anyway?